packages/common/src/api/tan-query/events/utils.ts

```diff
@@ -11,3 +20,8 @@ export const getEventListQueryKey = ({ pageSize }: { pageSize?: number }) => [
   QUERY_KEYS.events,
   { pageSize }
 ]
+
+export const getEventsByEntityIdQueryKey = (
+  entityId: ID | null | undefined,
+  options?: EventsByEntityIdOptions
+) => [QUERY_KEYS.eventsByEntityId, entityId, options]
```

Comment:
just a heads up, I just merged my changes that add typechecking for data by queryKey

please merge main and add types to all your queryKeys going forward

```suggestion
) => [QUERY_KEYS.eventsByEntityId, entityId, options] as unknown as QueryKey<InfiniteData<Entity[]>>
```